docs/reproducibility.md

```diff
@@ -10,53 +10,42 @@ You need a couple pieces of information to begin with:
     $ curl -s --cacert cacert.pem https://<LEDGER-URL>/app/version | jq ".scitt_version"
-    "0.7.2"
+    "1.2.3"
     ```
 
-## Reproduce measurement
+### Extract measurements
 
-To reproduce the same measurement do a docker build locally using the expected build image from [`microsoft/CCF`](https://github.com/microsoft/ccf):
+The quote contains the attestation report that has the necessary measurements. `Measurement` will have the launch measurement of the guest virtual machine (aka utility VM or UVM) which was used to run the container. `Host data` will have the security policy measurement which was used to make sure no other container was launched except the one you are inspecting, it will contain container image layer hashes.
 
-- Clone the repository and check out the tag or commit id that built the binary:
+- Decode the report and save it to a file:
 
-    ```
-    $ git clone ...
-    ...
-    $ git checkout 0.7.2
+    ```sh
+    cat node-quote.json | jq -r '.raw' | base64 -d > snp-report.bin
     ```
 
-- Identify the expected CCF build image version by inspecting the [Dockerfile](docker/enclave.Dockerfile) used for building the binary:
+- Inspect the report to get the measurements. You could use https://github.com/virtee/snpguest to display the report details:
 
-    ```
-    $ cat docker/enclave.Dockerfile | grep CCF_VERSION=
-    ARG CCF_VERSION=6.0.0-dev7
+    ```sh
+    snpguest display report snp-report.bin
     ```
 
-- Run a build inside of the CCF docker image and make sure to use a specific path (`__w/1/s`) to the sources as this is where our Azure build server copies the sources before building. If the build was done somewhere else, make sure to obtain the required path value:
+## Reproduce measurements
 
-    ```sh
-    $ export CCF_VERSION="6.0.0-dev7"
-    $ docker run -it --rm \
-        -w /__w/1/s -v $(pwd):/__w/1/s \
-        -v /var/run/docker.sock:/var/run/docker.sock \
-        --env PLATFORM=sgx \
-        FROM ghcr.io/microsoft/ccf/app/dev/sgx:ccf-"$CCF_VERSION" git config --global --add safe.directory "*" && ./docker/build.sh
-    ```
-- The build will print the value of `MRENCLAVE` in the output, similar to:
+### Guest VM
 
-    ```sh
-    mrenclave.txt
-    96c40e2532ba329849e7fede3f3d888a0423a1dc0f3d0511b138617cc3aa9e94
-    ```
+The details of how to reproduce the Guest VM are not ready yet. The source code will be open sourced at some point and it will be possible to build from it to reproduce the `measurement` in the report. It will make sure that the security policy enforcement logic is as expected.
+
+### Security policy and container image
 
-- As you can see in the example, the value printed in the build output matched the one from a running application.
\ No newline at end of file
+`Host data` contains the hash of the security policy, so we should be able to:
+- (TODO) build the docker image and create the same policy from it, or
+- (TODO) download the policy using the HTTP API, once it is available
```

Comment:
I would honestly just remove this part as it contains confusing information. We can populate it once we agree on the exact policy reproducibility steps on SNP.